pkg/model/components/docker.go

```diff
@@ -48,7 +48,9 @@ func (b *DockerOptionsBuilder) BuildOptions(o interface{}) error {
 
 	// Set the Docker version for known Kubernetes versions
 	if fi.StringValue(clusterSpec.Docker.Version) == "" {
-		if b.IsKubernetesGTE("1.17") {
+		if b.IsKubernetesGTE("1.18") {
+			docker.Version = fi.String("19.03.7")
```

Comment:
I couldn't find the note about this being the correct version.  Is this our decision or an upstream recommendation?

Reply:
The upstream only recommends Docker "19.03" in the release notes for 1.18. This is also why I proposed to use 19.03.8 for 1.17 in #8738.